checker/jdk/nullness/src/java/util/concurrent/ConcurrentHashMap.java

```diff
@@ -0,0 +1,42 @@
+package java.util.concurrent;
+import org.checkerframework.dataflow.qual.Pure;
+import org.checkerframework.dataflow.qual.SideEffectFree;
+import java.util.*;
+import org.checkerframework.checker.nullness.qual.KeyFor;
+import org.checkerframework.checker.nullness.qual.NonNull;
+import org.checkerframework.checker.nullness.qual.Nullable;
+
+public class ConcurrentHashMap<K extends @NonNull Object, V extends @NonNull Object> extends AbstractMap<K, V>
+        implements ConcurrentMap<K, V>, java.io.Serializable {
+
+    private static final long serialVersionUID = 0;
+
+     public ConcurrentHashMap(int initialCapacity,float loadFactor, int concurrencyLevel) {throw new RuntimeException("skeleton method");}
+     public ConcurrentHashMap(int initialCapacity, float loadFactor) {throw new RuntimeException("skeleton method");}
+     public ConcurrentHashMap(int initialCapacity) {throw new RuntimeException("skeleton method");}
+     public ConcurrentHashMap() {throw new RuntimeException("skeleton method");}
+     @Pure public boolean isEmpty() {throw new RuntimeException("skeleton method");}
+     @Pure public int size() {throw new RuntimeException("skeleton method");}
+     @Pure public @Nullable V get(@NonNull Object key){throw new RuntimeException("skeleton method");}
+     @Pure public boolean containsKey(@NonNull Object key) {throw new RuntimeException("skeleton method");}
+     @Pure public boolean containsValue(@NonNull Object value) {throw new RuntimeException("skeleton method");}
+     @Pure public boolean contains(@NonNull Object value) {throw new RuntimeException("skeleton method");}
+     public @Nullable V put(@NonNull K key,@NonNull V value) {throw new RuntimeException("skeleton method");}

```

Comment:
I wrote another simpler example to demonstrate this.

import org.checkerframework.checker.nullness.qual.*;
class A <T extends @NonNull Object> {
    
    void check (T var){
     System.out.println(var);   
    }
    
    public static void main(String[] args){
        
        A<@Nullable String> a=new A<>();
        a.check(null);
        
    }
    
}

Error: [type.argument.type.incompatible] incompatible types in type argument.
  found : @Initialized @Nullable String
  required: @Initialized @NonNull Object
line 10